doc/devdocs/ast.rst

```diff
@@ -183,12 +183,65 @@ These symbols appear in the ``head`` field of ``Expr``\s in lowered form.
 
     ``:pop_loc``: returns to the source location before the matching ``:push_loc``.
 
-LambdaInfo
+
+Method
+~~~~~~
+
+A unique'd container describing the shared metadata for a single (unspecialized) method.
```

Comment:
the past-particible of the verbification of `unique` (e.g. see the LLVM LangRef http://llvm.org/docs/LangRef.html)